### PR TITLE
Improve multiflatmap start up.

### PR DESCRIPTION
**Pull request by alan-wu**

### CODE REVIEW

File: src/components/viewers/MultiFlatmap.vue [hunk 1]
```diff
@@ -220,82 +220,95 @@ export default {
...
+          checkMarkersAtZoomLevel(flatmapImp, markers, this.zoomLevel);
+          this.restoreFeaturedMarkers(flatmapImp);
+        }
       }
     },
     getFlatmapImp: function () {
-      if (this.entry.type === "Flatmap") {
-        return this.$refs.flatmap.mapImp;
-      } else if (this.entry.type === "MultiFlatmap") {
+      if (this.entry.type === "MultiFlatmap") {
```
Comment (Tehsurfer) [inline src/components/viewers/MultiFlatmap.vue:255]: Does the normal flatmap not have a viewer anymore?

File: package.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
     "@abi-software/flatmapvuer": "^0.5.4",
     "@abi-software/map-side-bar": "^1.4.1-beta.1",
     "@abi-software/plotvuer": "^0.3.9",
-    "@abi-software/scaffoldvuer": "^0.2.1",
+    "@abi-software/scaffoldvuer": "^0.2.2",
```
Comment (Tehsurfer) [inline package.json:39]: was the scaffoldvuer change related to this?

**Review by Tehsurfer — APPROVED** [review]
Looks like this is switching to only using multiflatmapvuer for the mapimp. 

I'm guessing we don't use the singular flatmapvuer in mapintegratedvuer anymore?

**Comment by Tehsurfer** [comment]
I ran this locally, it worked as I would expect where I tested it.

I couldn't notice any speed increases though

Took me about 12s without cash, 10s with cache locally with about 3s variance

I get pretty similar times on sparc.science

File: src/components/viewers/MultiFlatmap.vue (hunk 1 above)
Comment (alan-wu) [reply src/components/viewers/MultiFlatmap.vue:255]: There is and is handled in Flatmap.vue. This should have been removed when I did the refactoring.

**Comment by alan-wu** [comment]
> I ran this locally, it worked as I would expect where I tested it.
> 
> I couldn't notice any speed increases though
> 
> Took me about 12s without cash, 10s with cache locally with about 3s variance
> 
> I get pretty similar times on sparc.science

It is not about the speed, it is more about starting the map and getting the information in them in a more reliable way.

File: package.json (hunk 2 above)
Comment (alan-wu) [reply package.json:39]: There are some changes related to a Portal ticket in it.

**Pull request merged**